assets/src/scripts/components/floodSlider.js

```diff
@@ -0,0 +1,60 @@
+const { select } = require('d3-selection');
+const { createStructuredSelector } = require('reselect');
+
+const { dispatch, link, provide } = require('../lib/redux');
+const { Actions } = require('../store');
+
+const updateSlider = function(node, {stages, gageHeight}) {
+    if (stages.length === 0) {
+        node.property('hidden', true);
+    } else {
+        const heightIndex = stages.indexOf(gageHeight);
+        node.select('input')
+            .attr('min', 0)
+            .attr('max', stages.length - 1)
+            .attr('step', 1)
+            .attr('value', heightIndex)
+            .attr('aria-valuemin', 0)
+            .attr('aria-valuemax', stages.length - 1)
+            .attr('aria-valuenow', heightIndex)
+            .attr('aria-valuetext', gageHeight);
+        node.select('.range-value').text(gageHeight);
+        node.property('hidden', false);
+    }
+};
+
+const floodSlider = function(node) {
+    const SLIDER_ID = 'fim-slider';
+    let sliderContainer = node.append('div')
+        .attr('class', 'slider-wrapper')
+        .property('hidden', true);
+    sliderContainer.append('label')
+        .attr('for', SLIDER_ID)
+        .html(`Gage Height: <output class="range-value" for="${SLIDER_ID}"></output> ft`);
```

Comment:
I don't think I've never seen the output tag used before! Why use `html()` rather than append a node?

Reply:
I hadn't either until I looked at how to make a slider accessible and there example used an output tag for the value. On the question, I just wanted the value in the output tag and wasn't sure how to get the text around it. Any suggestions?

Reply:
Oh, now that you say that, I don't know... I'd guess `text()` but haven't tried.